### PR TITLE
Add branching practices

### PR DESCRIPTION
**Pull request by rogeruiz**
This came out of our quarterly security review meeting today.
cc: :cloud:.gov team

### CODE REVIEW

**Comment by brittag** [comment]
Thanks for updating this! Added a commit with a few adjustments:

* Forking and branching are both fine.
* Add a hidden compliance note - that it's fine for us to adjust the GitHub Guidelines section without major concerns about compliance, as long as the guidelines are within our formal requirements.
* GitHub built-in review requests are fine.

**Review by rogeruiz — COMMENTED** [review]
Thanks @brittag. I can't approve my own PR, but this is great. Thank you!

**Review by brittag — APPROVED** [review]

File: content/docs/ops/configuration-management.md [hunk 1]
```diff
@@ -54,13 +54,24 @@ If possible, Configuration Management tools need to be set up to always roll bac
...
+<!-- As long as these GitHub guidelines comply with the FedRAMP-required policies above and our 18F GitHub requirements, we can adjust them at will, according to our team preferences. -->
+
 As part of 18F, we follow [the 18F requirements for using GitHub](https://handbook.18f.gov/github/). These are our team practices within those requirements.
 
 ### Forking vs. branching
 
-The team prefers forking.
+Both forking and branching are welcome in our repositories. Contributors inside 18F can use forking or branching according to their personal preferences, and contributors outside 18F can fork repositories.
+
+The team often practices branching. The rationale for practicing branching is that when pairing, collaborating on a
```
Comment (jameshupp) [inline content/docs/ops/configuration-management.md:65]: I'd get rid of some -ing words in this paragraph. Maybe word it like this? `The team often practices branching. The rationale for branching within a team is that paired collaboration on a single branch avoids certain types of friction:`

Comment (jameshupp) [reply content/docs/ops/configuration-management.md:65]: Whoops! Too late?

Comment (brittag) [reply content/docs/ops/configuration-management.md:65]: Too late! Totally welcome to open a new PR though! A bunch of this page could use some smoothing out if we feel like it.

Comment (rogeruiz) [reply content/docs/ops/configuration-management.md:65]: never too late! sorry @jameshupp we can totally update it with another PR. 😬

**Pull request merged**